File: src/main/java/es/pablouser1/umacraft/models/EmailConfig.java
package es.pablouser1.umacraft.models;

public class EmailConfig {
    public String host;
    public int port;
    public String username;
    public String password;
    public String encryption;

    public EmailConfig(String host, int port, String username, String password, String encryption) {
        this.host = host;
        this.port = port;
        this.username = username;
        this.password = password;
        this.encryption = encryption;
    }
}

File: src/main/kotlin/es/pablouser1/umacraft/models/EmailConfig.kt
package es.pablouser1.umacraft.models

import es.pablouser1.umacraft.enums.EmailEncryptionEnum

data class EmailConfig(
    val host: String,
    val port: Int,
    val username: String,
    val password: String,
    val encryption: EmailEncryptionEnum,
)
